app/build.gradle.kts

```diff
@@ -66,4 +80,19 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+
+    // Dagger Hilt
+    implementation("com.google.dagger:hilt-android:2.49")
+    kapt("com.google.dagger:hilt-android-compiler:2.46")
+    // Hilt Navigation Compose
+    implementation("androidx.hilt:hilt-navigation-compose:1.2.0")
+
+    // Moshi
+    implementation("com.squareup.moshi:moshi-kotlin:1.15.0")
+
+    // Retrofit
+    implementation("com.squareup.retrofit2:retrofit:2.9.0")
+
+    // Moshi + Retrofit
+    implementation("com.squareup.retrofit2:converter-moshi:2.4.0")
```

Comment:
Which IDE version did you create this project on? Nowadays, the default project should use version catalogs for dependency management, have you looked into that?

Reply:
I am using HedgeHog 2023.1.1, but yes I will update to Koala 2024.1.1, and move dependency version to version catalogs